### ISSUE TITLE
Mentoring Issue

### ISSUE THREAD

**Issue by infamousdegen** [exercism/exercism#5919]
Hello , 

When I try to request for a mentor it shows mentoring tracks for me are full but I don't have any open mentor discussion going on ,

**Comment by angelikatyborska** [comment]
We will need your Exercism handle to check this. Is it the same as your GitHub handle (`infamousdegen`)?  For which tracks are you observing this problem? Is it a problem on all the tracks that you're using (if you're using more than one)?

**Comment by infamousdegen** [comment]
Yep its the same infamousdegen . I am observing this problem for python.
